src/App.tsx

```diff
@@ -186,37 +187,31 @@ export default function App() {
             .join('')};
         </div> */}
       {choices?.length > 0 && (
-        <div
-          ref={scrollRef}
+        <SimpleBar
+          scrollableNodeProps={{ ref: scrollRef }}
+          className="px-4 pb-4 flex flex-col text-black dark:text-white w-full max-h-screen focus:border-none focus:outline-none outline-none"
```

Comment:
adding padding to the scroll view and then border radius for action items and a little gap between them takes it up a notch